packages/blade/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -22,11 +22,12 @@ export default {
 const TabsTemplate: ComponentStory<typeof Tabs> = () => {
   return (
     <Box height="100%">
-      <Tabs variant="bordered" isFullWidthTabItem={false}>
+      <Tabs variant="bordered" isFullWidthTabItem={true}>
```

Comment:
Lots of unused components here @anuraghazra

Reply:
All removed in Docs PR.